client/blocks/nps-survey/index.jsx

```diff
@@ -38,75 +40,83 @@ class NpsSurvey extends Component {
 
 	handleFinishClick = () => {
 		this.props.submitNpsSurvey( this.props.name, this.state.score );
+		this.onClose( this.showThanksNotice );
 	}
 
 	handleDismissClick = () => {
 		this.props.submitNpsSurveyWithNoScore( this.props.name );
-		// allow for the state to propagate
+		this.onClose( noop );
+	}
+
+	showThanksNotice = () => {
+		this.props.successNotice(
+			this.props.translate( 'Thanks for your feedback!' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Thanks for your feedback.', { context: 'calypso stats survey'} )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).